plugin/evm/vm.go

```diff
@@ -550,13 +550,9 @@ func (vm *VM) Initialize(
 }
 
 func (vm *VM) initializeMetrics() error {
+	metrics.Enabled = true
```

Comment:
can we add few comments why we explicitly enable it here?

Reply:
Added

```go
	// [metrics.Enabled] is a global variable imported from go-ethereum/metrics
	// and must be set to true to enable metrics collection.
```

Does that suffice? 🤔